recipes/sqlite/sqlite3.inc

```diff
@@ -25,18 +37,21 @@ EXTRA_OECONF_READLINE = "--disable-readline"
 EXTRA_OECONF_READLINE:USE_sqlite3_readline = "--enable-readline"
 EXTRA_OECONF += "${EXTRA_OECONF_READLINE}"
 DEPENDS:>USE_sqlite3_readline = " libreadline"
+DEPENDS_sqlite3-sqlite3:>USE_sqlite3_readline = " libreadline"
```

Comment:
think it is a bit more "clean" and like we do elsewhere, to add these to the AUTO_PACKAGE_*_{R}DEPENDS vars instead